.github/workflows/deploy.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         run: npm install && npm run build
 
       - name: Deploy app build to S3 bucket
+        run: mv ./out/[[...app]].html ./out/index.html
```

Comment:
You'll need to format this as a multi-line command
e.g.
```
run: |
  mv ./out/[[...app]].html ./out/index.html
  aws s3 sync ./out/ s3://${{ secrets.AWS_S3_BUCKET_PREPROD }} --delete
```

Reply:
ah nice one @Judge40 🌮